pom.xml

```diff
@@ -47,4 +48,18 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+
+    <distributionManagement>
```

Comment:
this should be here? this way connectors that use this parent will have the same DM